src/v2/Apps/Auction/Components/BidForm.tsx

```diff
@@ -289,16 +280,16 @@ export const BidForm: React.FC<Props> = ({
                       {status}.
                     </Sans>
                   )}
-
-                  <Button
-                    size="large"
-                    width="100%"
-                    loading={isSubmitting}
-                    {...({ type: "submit" } as any)}
-                  >
-                    Confirm bid
-                  </Button>
                 </Flex>
+
+                <Button
+                  size="large"
```

Comment:
same question here, i trust all is fine but element seems to have left its container